dev-libs/spdlog/files/spdlog-unbundle-fmt.patch

```diff
@@ -1,90 +1,37 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Changing the patch in place will break older versions. While at it, please include version in the patch filename.

Reply:
What do you mean with "in place"? Also, when a live ebuild is available is needed to include the version on the patch filename? In that case it should be hardcoded on the live ebuild?
I removed the index of the patch as it's not needed

Reply:
What I mean is that when you change a patch in an incompatible way (i.e. that existing ebuilds would behave differently), you should copy it to a new file. Normally we use package version for that.

For example, you'd name the new patch `spdlog-0.17.0-unbundle-fmt.patch` and use that name in all future ebuilds. Then, when e.g. you need to change it for 0.20.0, you'd create a new `spdlog-0.20.0-unbundle-fmt.patch` and start using that. Old patches would be removed when the last ebuild using them is removed.

Reply:
Although it's not strictly necessary (because both patches are the same) I created two patches with their version numbers, this way I could use ${P} directly on both ebuilds without hardcoding the 0.17.0 patch on the live ebuild. I hope that this is the correct way.

Reply:
Nope, it isn't. Duplicate files are undesired. People will get confused over whether they're different.

Reply:
It should be correct now

Reply:
Nope, the first commit is still the same.

Reply:
... yep, I've screwed it up a little bit, I've made the change on the clean commit instead of the verbump one. It should be okay now

Reply:
You're still removing the old patch too early.